internal/controllers/clusterrequest_controller.go

```diff
@@ -104,6 +125,37 @@ func (t *clusterRequestReconcilerTask) run(ctx context.Context) (nextReconcile c
 	// Set the default reconcile time to 5 minutes.
 	nextReconcile = ctrl.Result{RequeueAfter: 5 * time.Minute}
 
+	// Check if the ClusterInstance has been already created.
+	if t.object.Status.ClusterInstanceStatus != nil {
+		// If the provisioning has started and the managed cluster is created, there are no
+		// further changes that can be done to the ClusterInstance, so we just update the status.
+		// To make changes to the managed cluster, the ClusterRequest needs to be deleted
+		// and recreated.
```

Comment:
Won't deleting the ClusterRequest result in the managed cluster being deleted as well? The comment seems to indicate it would just be modified.